app/backend/.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+default_language_version:
```

Comment:
I don't like pre-commit hooks personally, but do not mind it being in the repo for others if it's not automatic. Is this optional?

Reply:
Yes, this only is applied if you run `pre-commit install` at some times, otherwise it won't do anything. I was wondering if I put it into the root of the repository as it will run for the whole repository in general, but then again it is fairly specific to python and the backend code. What do you think?

Reply:
OK sounds good. Would you like to add some docs on how to enable ti?

Reply:
I added a paragraph to the contributing Readme, have a look if it is self-explanatory :)